client/components/payment-activity/style.scss

```diff
@@ -1,15 +1,55 @@
 .wcpay-payments-activity {
 	&__card {
-		padding: 24px 24px 16px 24px;
-
 		&__header {
-			padding: 0 !important;
-			margin-bottom: 32px;
+			padding: 16px 24px !important;
+			font-size: 20px;
+			font-weight: 400;
 		}
 
 		&__body {
 			text-align: center;
-			color: $studio-gray-30;
+			padding: 0 !important;
+		}
+	}
+}
+
+.payments-activity-data {
+	display: flex;
+	padding: 24px;
+	border-top: 1px solid $gray-200;
+	border-bottom: 1px solid $gray-200;
+
```

Comment:
⛏️ Nit: we should remove these border-top/bottom CSS declarations, as they are duplicating borders that are handled by CardHeader and CardFooter (Emoji Survey).
```suggestion
```

Reply:
Thanks @Jinksi! Done.